bin/dev

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
```

Comment:
To my previous comment, these 4 lines will install foreman if it's not on the machine, so this is why you don't need it in the `Gemfile`.

Reply:
Got it. Foreman is useful to keep multiple processes in control. Sort of a Docker compose. This way I could even be developing with local Redis and Postgress. Neat.